client/my-sites/site-settings/form-general.jsx

```diff
@@ -442,6 +444,46 @@ const FormGeneral = React.createClass( {
 		);
 	},
 
+	helloVoteOption() {
```

Comment:
In 0bc2cc2 I changed the option text to match the notice in #8423, it should be a bit more clear now.